src/tools/dartpad.md

```diff
@@ -13,6 +13,16 @@ Here's what DartPad looks like:
 <img src="images/DartPadWindow.png" alt="DartPad screenshot" />
 
 
+## Library support
+
+DartPad supports [dart:* libraries]({{site.dart_api}}) that work with web apps;
+it doesn't support [dart:io]({{site.dart_api}}/stable/dart-io) or
+libraries from [packages.]({{site.pub}})
```

Comment:
nit: "libraries from packages" --> "libraries from pub.dartlang.org"? I know it's technically less correct but "libraries from packages" sounds like "programs from files" to me.

Not feeling strongly about this, feel free to ignore.

Reply:
I started implementing this, but I couldn't figure out how to make the next sentence work well if I didn't introduce "packages" here.